INSTALL.rst

```diff
@@ -460,8 +460,8 @@ the following commands.
 
     (invenio)$ inveniomanage config set CFG_EMAIL_BACKEND flask_email.backends.console.Mail
     (invenio)$ inveniomanage config set CFG_BIBSCHED_PROCESS_USER $USER
-    (invenio)$ inveniomanage config set CFG_DATABASE_NAME $BRANCH
-    (invenio)$ inveniomanage config set CFG_DATABASE_USER $BRANCH
+    (invenio)$ inveniomanage config set CFG_DATABASE_NAME `echo $BRANCH | sed -e 's/[^A-Za-z0-9]//g'`
```

Comment:
can we create better some cleaned name after https://github.com/inveniosoftware/invenio/pull/3304/files#diff-26dee18463b5da190f343abcc74cf708R343?

--
PS: Can you rebase this patch for `maint-2.0`?